runtime/compiler/env/j9methodServer.cpp

```diff
@@ -2054,6 +2054,33 @@ TR_ResolvedJ9JITServerMethod::archetypeArgPlaceholderSlot()
    return paramSlots;
    }
 
+bool
+TR_ResolvedJ9JITServerMethod::isFieldQType(int32_t cpIndex)
+   {
+   if (!TR::Compiler->om.areValueTypesEnabled() ||
+      (-1 == cpIndex))
+      return false;
+
+   auto comp = _fe->_compInfoPT->getCompilation();
+   int32_t sigLen;
+   char *sig = fieldOrStaticSignatureChars(cpIndex, sigLen);
```

Comment:
Doing that would require also adding versions of `getROMString` and `getRemoteROMString` that return UTF8, since `fieldOrStaticSignature` uses them to get the signature. 
If that's something worth doing maybe it's better to do it in a separate PR?

Reply:
I think it's worth doing, so let's open another PR for that